addons/addon-base-raas/packages/base-raas-services/lib/schema/create-study.json

```diff
@@ -32,12 +32,6 @@
     "uploadLocationEnabled": {
       "type": "boolean"
     },
-    "sha": {
```

Comment:
Do we need to change OpenAPI documentation

Reply:
updated the OpenApi docs